README-dockerhub.md

```diff
@@ -113,14 +115,13 @@ jobs:
           AQUA_SECRET: ${{ secrets.AQUA_SECRET }}
           GITHUB_TOKEN: ${{ github.token }}
           TRIVY_RUN_AS_PLUGIN: "aqua"
-          # For proxy configuration add env vars: HTTP_PROXY/HTTPS_PROXY, CA-CRET (path to CA certificate)
+          # Use here any other environment variable
 ```
 
 ### Usage for running manually using docker command
 
 ```bash
-EXPORT AQUA_KEY=${AQUA_KEY} AQUA_SECRET=${AQUA_SECRET}
-docker run -it aquasec/aqua-scanner trivy fs --scanners config,vuln,secret .
+AQUA_KEY=${AQUA_KEY} AQUA_SECRET=${AQUA_SECRET} TRIVY_RUN_AS_PLUGIN=aqua docker run -it -e AQUA_KEY -e AQUA_SECRET -e INPUT_WORKING_DIRECTORY=/scanning -v "${YOUR_WORKSPACE}":"/scanning" aquasec/aqua-scanner trivy fs --scanners config,vuln,secret .
```

Comment:
Not sure but need to check the following:
1. is the /scanning dir created or does the user need to create it? (its not part of the image)
2. it might be problematic with our limited image, see: https://github.com/aquasecurity/trivy-commercial/blob/069351549686a62d40e29920cebd1df9f578f08e/Dockerfile.limited#L79-L80

Reply:
1. No need to create the dir, it is mounted (checked)
2. Take a look at the entrypoint (https://github.com/aquasecurity/trivy-commercial/blob/069351549686a62d40e29920cebd1df9f578f08e/entrypoint.sh) --> Validated that both of them are working